i3pystatus/openvpn.py

```diff
@@ -46,6 +47,7 @@ class OpenVPN(IntervalModule):
         ("status_down", "Symbol to display when down"),
         ("status_up", "Symbol to display when up"),
         ("vpn_name", "Name of VPN"),
+        ("openvpn_lastest", "Use lastest openvpn service names"),
```

Comment:
Looks good, and this is a bit picky, but do think something like `detect_service_syntax` might be a better name?